workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/dashboard/LevelChart/components/PartnershipLevel/PartnershipLevel.tsx

```diff
@@ -60,12 +60,12 @@ const PartnershipLevel = ({
 	partnerLevel,
 }: IPropsPartnershipLevel) => {
 	const getTotalARR = () => {
-		if (partnerLevel.partnerLevelType.key === PartnershipLevels.GOLD) {
-			return partnerLevelProperties[partnerLevel.partnerLevelType.key]
-				.goalARR;
+		if (partnerLevel.partnerLevelType.key === PartnershipLevels.PLATINUM) {
+			return aRRResults.targetArr;
 		}
 
-		return aRRResults.aRRAmountTotal;
+		return partnerLevelProperties[partnerLevel.partnerLevelType.key]
+			.goalARR;
```

Comment:
Melhor manter a logica original aqui.

```suggestion
		if (partnerLevel.partnerLevelType.key === PartnershipLevels.GOLD) {
			return partnerLevelProperties[partnerLevel.partnerLevelType.key]
				.goalARR;
		}

		return aRRResults.aRRAmountTotal;
```